app/src/main/java/org/wikipedia/readinglist/database/ReadingList.kt

```diff
@@ -19,35 +19,22 @@ class ReadingList(var dbTitle: String,
     @Transient
     private var accentAndCaseInvariantTitle: String? = null
 
-    val isDefault = dbTitle.isEmpty()
+    val isDefault: Boolean = title.isEmpty()
 
-    val title = if (isDefault) WikipediaApp.getInstance().getString(R.string.default_reading_list_name) else dbTitle
+    var title = title
+        get() = if (isDefault) WikipediaApp.getInstance().getString(R.string.default_reading_list_name) else field
 
-    val numPagesOffline: Int
-        get() {
-            var count = 0
-            for (page in pages) {
-                if (page.offline && page.status == ReadingListPage.STATUS_SAVED) {
-                    count++
-                }
-            }
-            return count
-        }
+    val numPagesOffline
+        get() = pages.count { it.offline && it.status == ReadingListPage.STATUS_SAVED }
 
     val sizeBytesFromPages: Long
-        get() {
-            var bytes = 0L
-            pages.forEach {
-                bytes += if (it.offline) it.sizeBytes else 0
-            }
-            return bytes
-        }
+        get() = pages.sumOf { if (it.offline) it.sizeBytes else 0 }
```

Comment:
loving these convenience constructs.

Reply:
You are correct. Not sure why these didn't work for me at the moment when doing the conversion. The size numbers will not show the expected values.